README.md

```diff
@@ -111,6 +111,66 @@ When running on cluster node, you will also need to distribute this keytab, belo
 
 When Spark application is started, it will transparently track the execution plan of submitted SQL/DF transformations, parse the plan and create related entities in Atlas.
 
+Spark models vs Hive models
+====
+
+SAC classifies table related entities with two different kind of models: Spark / Hive.
+
+We decided to determine which models SAC should use via taking realistic condition: whether Hive Hook can take care of table related entities or not. For creating Hive entities, SAC is not the best one to fill out information on entities - so we let Hive to do its own.
```

Comment:
SAC should use via taking realistic condition -> SAC should create based on ?
For creating Hive entities, SAC is not the best one to fill out information on entities - so we let Hive to do its own -> Somthing like "For Hive entities, SAC relies on HMS Atlas hook as the source of truth and expects HMS atlas hook to create it."

Reply:
Much nicer representation. Thanks!